client/go/cmd/login.go

```diff
@@ -27,7 +37,57 @@ func newLoginCmd(cli *CLI) *cobra.Command {
 			if err != nil {
 				return err
 			}
-			_, err = auth0.RunLogin(ctx, a, false)
+			state, err := a.Authenticator.Start(ctx)
+			if err != nil {
+				return fmt.Errorf("could not start the authentication process: %w", err)
+			}
+
+			fmt.Printf("Your Device Confirmation code is: %s\n\n", state.UserCode)
+
+			fmt.Println("If you prefer, you can open the URL directly for verification")
+			fmt.Printf("Your Verification URL: %s\n\n", state.VerificationURI)
+
+			fmt.Println("Press Enter to open the browser to log in or ^C to quit...")
+			fmt.Scanln()
```

Comment:
These should print to `cli.Stdout` or just use `log.PrintXXX`. Same below.

Reply:
Fixed, but we are left with the ugly fmt.Scanln()` duckling. But I leave that to be handled as part of the ongoing discussion we have about the UX of this command.

Reply:
`fmt.Fscanln(cli.Stdin)` should work.